initializers/tasks.js

```
@@ -540,10 +542,19 @@ module.exports = class Tasks extends ActionHero.Initializer {
 
     api.tasks.loadTasks = (reload) => {
       api.config.general.paths.task.forEach((p) => {
-        api.utils.recursiveDirectoryGlob(p).forEach((f) => {
+        glob.sync(path.join(p, '**', '*.js')).forEach((f) => {
           api.tasks.loadFile(f, reload)
         })
       })
+
+      for (let pluginName in api.config.plugins) {
```

A little confused about a detail - you may know this instinctively because you just wrote it. What's the order of operations here if your main project overrides a file in the plugin, as it can override core today? Will that still work? It SEEMS like this is backward to support that but I wasn't sure at first glance and it's late...

---

good question! You can't override initializers in core (it throws an error), and there are no actions or tasks which core exports (just a few it dumps into a new project on `generate`).  So... there really was no thought at all given to overriding things.  Should we throw an error?  Should we allow overriding?

---

I like the idea of allowing overriding but logging a warning message.